jetson/ra_kinematics/mrover_arm.hpp

```diff
@@ -227,10 +239,35 @@ class MRoverArm {
 
     void check_joint_limits(std::vector<double> &angles);
 
-    double joint_b_stabilizer(double angle);
-
     void send_kill_cmd();
 };
 
+class StandardArm : public MRoverArm {
```

Comment:
Make publish_config, send_kill_cmd virtual functions as well (and others if they make sense).